public/index.html

```diff
@@ -23,7 +23,7 @@
     <header>
       <!-- Menu Bar [File][Character!][Inventory!][Map!][Spells][Activate][Verbs][Window][Help] -->
       <nav class="navbar navbar-default menu">
-        <div class="container-fluid">
```

Comment:
sorry, when I commented about this i meant I don't like the class `container`   but `container-fluid` is okay and useful I think we should add it back.